docs/admin/cluster-management.md

```diff
@@ -91,7 +91,7 @@ If your cluster runs short on resources you can easily add more machines to it i
 If you're using GCE or GKE it's done by resizing Instance Group managing your Nodes. It can be accomplished by modifying number of instances on `Compute > Compute Engine > Instance groups > your group > Edit group` [Google Cloud Console page](https://console.developers.google.com) or using gcloud CLI:
 
 ```
-gcloud compute instance-groups managed --zone $ZONE resize my-cluster-minon-group --new-size 42
+gcloud compute instance-groups managed resize kubernetes-minion-group --size 42
```

Comment:
Let's keep `--zone` to disambiguate in certain cases. I checked and --zone needs to be moved closer to the end of the command (`gcloud compute instance-groups managed resize NAME --size SIZE [--zone ZONE] [GLOBAL-FLAG ...]`).

```gcloud compute instance-groups managed resize kubernetes-minion-group --size 42 --zone $ZONE```